pystac/collection.py

```diff
@@ -9,6 +12,16 @@
 from pystac.utils import datetime_to_str
 
 
+fieldsfilename = os.path.join(os.path.dirname(__file__),
+                              "resources", "fields-normalized.json")
```

Comment:
Would it make sense to load it from a CDN by default? That file is expected to change pretty often (mostly to add new fields)...